File: 2748-calculate-delayed-arrival-time/calculate-delayed-arrival-time.cpp
class Solution {
public:
    int findDelayedArrivalTime(int arrivalTime, int delayedTime) {
        int reachingTime = arrivalTime + delayedTime;
        int value;

        if(reachingTime < 24)
        {
            value = reachingTime;
        }
        else if(reachingTime == 24)
        {
            value = 0;
        }
        else if(reachingTime > 24){
            value = reachingTime - 24;
        }
        return value;
    }
};

File: 2748-calculate-delayed-arrival-time/calculate-delayed-arrival-time.py
class Solution:
    def findDelayedArrivalTime(self, arrivalTime: int, delayedTime: int) -> int:
        totalTime = arrivalTime + delayedTime
        if (totalTime == 24):
            return 0
        elif (totalTime < 24):
            return totalTime
        else:
            return (totalTime - 24)
        